src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
Ça plane pour moi. We can always adjust it later.

@leemunroe does that work for you?